egs/fame/v1/local/make_fame_train.pl

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/perl
+#
+# Copyright 2015   David Snyder
+#           2017   Radboud University (Author: Emre Yilmaz) 
+# Apache 2.0.
+# Usage: make_sre_2010_train.pl /export/corpora5/SRE/SRE2010/eval/ data/.
```

Comment:
Please be sure to update these comments. Right now it refers to make_sre_2010_train.pl when it should be make_fame_train.pl.

Reply:
Fixed..